src/GraphQL.Serialization.Newtonsoft/Http/NewtonsoftContractResolver.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace GraphQL.Http
+{
+    internal class NewtonsoftContractResolver : DefaultContractResolver
+    {
+        private readonly IContractResolver _contractResolver;
```

Comment:
No need in inner resolver, see other comments.

Reply:
Removed